tests/textcase/lsp/lsp_spec.lua

```diff
@@ -1 +1,62 @@
-describe("LSP", function() end)
+local test_helpers = require("tests.test_helpers")
+local textcase = require("textcase")
+
+local cur_dir = vim.fn.expand("%:p:h")
+
+---@param path string
+function M.read_file(path)
+  local fd = vim.loop.fs_open(path, "r", 438)
+  local fstat = vim.loop.fs_fstat(fd)
+  local contents = vim.loop.fs_read(fd, fstat.size, 0)
+  vim.loop.fs_close(fd)
+  return contents
+end
+
+function M.waitFor(max_time, callback)
+  local max_seconds = max_time
```

Comment:
I think this is an unnecessary assignment. Just call the function param `max_seconds`.